src/views/Events.vue

```diff
@@ -73,56 +75,32 @@ export default {
       }
       return currentEvents;
     },
+
+    //codeIsValid: returns true iff the code is valid.
+    //Needs testing
     codeIsValid() {
-      // In future use regex to prevent leading zeros?
+      //TICKET: Use regex to prevent leading zeros?
```

Comment:
We're probably gonna replace the codes entirely with something else, maybe like two randomly generated words. This is fine for now though, just an FYI.